examples_tests/18.MitsubaLoader/main.cpp

```diff
@@ -260,6 +260,23 @@ int main()
 		params.WindowSize.Width = film.width;
 		params.WindowSize.Height = film.height;
 	}
+	else return 1; // no cameras
+
+	const auto& sensor = globalMeta->sensors.front(); //always choose frist one
+	auto isOkSensorType = [](const ext::MitsubaLoader::CElementSensor& sensor) -> bool {
+		return sensor.type == ext::MitsubaLoader::CElementSensor::Type::PERSPECTIVE || sensor.type == ext::MitsubaLoader::CElementSensor::Type::THINLENS;
+	};
+
+	if (!isOkSensorType(sensor))
+		return 1;
+
+	bool leftHandedCamera = false;
```

Comment:
you need to see is the sensor is left or right handed (viewProj matrix can be checked for that, see example RTAO)

Reply:
u just commented the line doing this with flag inversion